src/modules/rlm_perl/rlm_perl.c

```diff
@@ -1053,8 +1053,12 @@ DIAG_ON(nested-externs)
 
 static int mod_load(void)
 {
+#define LOAD_INFO(_fmt, ...) fr_log_perror(LOG_DST, L_INFO, __FILE__, __LINE__, "rlm_perl - " _fmt,  ## __VA_ARGS__)
 #define LOAD_WARN(_fmt, ...) fr_log_perror(LOG_DST, L_WARN, __FILE__, __LINE__, "rlm_perl - " _fmt,  ## __VA_ARGS__)
 
+	LOAD_INFO("Perl version: %s", PERL_API_VERSION_STRING);
```

Comment:
Just use INFO, you're not printing an error, and you'll print irrelevant error messages from the error stack if they've not been cleared by previous code.

Reply:
done